src/sentry/tasks/statistical_detectors.py

```diff
@@ -84,12 +114,19 @@ def run_detection() -> None:
     queue="performance.statistical_detector",
     max_retries=0,
 )
-def detect_transaction_trends(project_ids: List[int], start: datetime, *args, **kwargs) -> None:
+def detect_transaction_trends(
+    org_ids: List[int], project_ids: List[int], start: datetime, **kwargs
+) -> None:
```

Comment:
```suggestion
def detect_transaction_trends(
    org_ids: List[int], project_ids: List[int], start: datetime, *args, **kwargs
) -> None:
```

Please make sure that `*args` is not removed here otherwise you could run into a transient error between deployments where the previous deploy dispatches a task and a new deploy picks up the task.

Also, most of the time, when changing the arguments to a task, it should be broken up into
- introduce the new argument as an optional parameter
- pass the new argument to the new task

But for this PR, I think it's fine to let the old tasks fail.

Reply:
done